client/bindings/python/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [workspace]
 
 [package]
-name = "iota-client-python"
```

Comment:
I would like to keep the name like that for consistency but we can merge for now and find a solution later. There is already an issue to sort these things out.